### PR TITLE
Adjust styles of `VContentLink`

### LINKED ISSUE #732: Styles in Content link component
## Description
The **Content link** component has the following style errors:

### Texts alignment
Texts are not vertically-aligned correctly. It seems that content label has a padding-top pushing the element.

CleanShot 2022-01-27 at 11 31 36@2x

### Results text style
The results text has an incorrect style. It should follow the `caption regular` style.

### Focus style
The `focus` style has an extra gray border. It should have only a pink border, as the [Figma component](https://www.figma.com/file/GIIQ4sDbaToCfFQyKMvzr8/Openverse-Design-Library?node-id=379%3A1124).

CleanShot 2022-01-27 at 11 35 15@2x

### LINKED ISSUE #778: Pink text underline on content links
## Description

VContentLink now has a pink underline on hover.

CleanShot 2022-02-01 at 14 44 00@2x

Most likely related to the styling of raw anchor tags in #727 

## Resolution

- [ ] 🙋 I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by krysal**
## Fixes

Fixes #732  by @panchovm 
Fixes #778 by @zackkrida 

## Description

This PR:
- Centers vertically the "See all <media>" label 
- Uses the correct font size for the results text
- Removes the underline and pink color on hover
- Changes breakpoint to when it change from mobile to desktop layout as we're mainly using `md` and around the 1000px of width it was still showing the mobile version

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`).
- [x] My pull request targets the *default* branch of the repository (`main`) or a parent feature branch.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [-] I added or updated tests for the changes I made (if applicable).
- [-] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no visible errors.

[best_practices]:https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by panchovm** [comment]
It looks great ✨

File: src/components/VAllResultsGrid/VAllResultsGrid.vue [hunk 1]
```diff
@@ -4,14 +4,14 @@
...
         v-for="[key, item] in results"
         :key="key"
-        :to="localePath({ path: `/search/${key}`, query: $route.query })"
-        class="lg:col-span-2 focus:bg-white focus:border-tx focus:ring focus:ring-pink focus:outline-none focus:shadow-ring focus:text-black rounded-sm"
-      >
-        <VContentLink :results-count="item.count" :media-type="key" />
-      </NuxtLink>
+        :media-type="key"
+        :results-count="item.count"
+        :to="{ path: `/search/${key}`, query: $route.query }"
```
Comment (zackkrida) [inline src/components/VAllResultsGrid/VAllResultsGrid.vue:12]:
```suggestion
        :to="{ path: `/search/${key}`, query: $route.query }"
```

This should be restored to use `localePath`, try visiting `http://localhost:8443/es/search/?q=dog` and observe the missing `/es` path on these routes.

**Review by zackkrida — CHANGES_REQUESTED** [review]
One bug with the urls, but looks excellent!

**Review by sarayourfriend — APPROVED** [review]
LGTM! Approving assuming the bug Zack saw will be fixed.

Nice fix up! 🤠

File: src/components/VAllResultsGrid/VAllResultsGrid.vue (hunk 1 above)
Comment (krysal) [reply src/components/VAllResultsGrid/VAllResultsGrid.vue:12]: Right! I don't know how that slipped out, thank you!

**Review by zackkrida — APPROVED** [review]

**Pull request merged**